rules/phpunit/tests/Rector/AssertEqualsToSameRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\AssertEqualsToSameRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+final class MyTest extends TestCase
+{
+    public function test()
+    {
+        $int = 1;
+        $expectedInt = 1;
+        $this->assertEquals($expectedInt, $int);
+
+        $float = 1.1;
+        $expectedFloat = 1.1;
+        $this->assertEquals($expectedFloat, $float);
+
+        $string = 'abc';
+        $expectedString = 'abc';
+        $this->assertEquals($expectedString, $string);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\AssertEqualsToSameRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+final class MyTest extends TestCase
+{
+    public function test()
+    {
+        $int = 1;
+        $expectedInt = 1;
+        $this->assertSame($expectedInt, $int);
+
+        $float = 1.1;
+        $expectedFloat = 1.1;
+        $this->assertSame($expectedFloat, $float);
+
+        $string = 'abc';
+        $expectedString = 'abc';
+        $this->assertSame($expectedString, $string);
+    }
+}
+
+?>
```

Comment:
How can be sure from tests that the null/bool cases are skipped?

Reply:
I found it quite clear from the specific type map.
But sure, I can add a skip one too.